### PR TITLE
travis: slight tweak

### PR DESCRIPTION
**Pull request by xu-cheng**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,7 +27,11 @@ before_install:
   - git clean -qxdff
 
 script:
-  - brew test-bot
+  - if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then
```
Comment (mikemcquaid) [inline .travis.yml]: Can we push this logic to the `test-bot` itself?

**Comment by mikemcquaid** [comment]
:+1: on the other changes

File: .travis.yml (hunk 1 above)
Comment (xu-cheng) [inline .travis.yml]: Done.

Thought, I feel like we should sort out these logics (but I have no clue what purpose for each branches are) https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/cmd/test-bot.rb#L302-L355

Comment (mikemcquaid) [inline .travis.yml]: Yeh, agreed. Basically they are so `test-bot` can use environment variables, a specified commit, a PR URL, a formula etc. as input.

**Comment by mikemcquaid** [comment]
:shipit: when :green_apple:

**Pull request closed without merging**